tools/symbol-check.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python3
+'''
+A script to check that a secp256k1 shared library
+exports only expected symbols.
+
+Example usage:
+
+- when building with Autotools:
+
+    ./tools/symbol-check.py .libs/libsecp256k1.so
+or
+    ./tools/symbol-check.py .libs/libsecp256k1-<V>.dll
+or
+    ./tools/symbol-check.py .libs/libsecp256k1.dylib
+
+- when building with CMake:
+
+    ./tools/symbol-check.py build/lib/libsecp256k1.so
+or
+    ./tools/symbol-check.py build/bin/libsecp256k1-<V>.dll
+or
+    ./tools/symbol-check.py build/lib/libsecp256k1.dylib
+'''
+import re
+import sys
+import subprocess
+from typing import List
+
+import lief
+
+
+def grep_exported_symbols() -> List[str]:
+    grep_output = subprocess.check_output(["git", "grep", "^SECP256K1_API", "--", "include"], universal_newlines=True, encoding="utf8")
```

Comment:
```suggestion
    grep_output = subprocess.check_output(["git", "grep", "^[\s*]SECP256K1_API", "--", "include"], universal_newlines=True, encoding="utf8")
```
That's perhaps a bit more robust